tests/integration/app-charm/metadata.yaml

```diff
@@ -15,5 +15,9 @@ peers:
     interface: cluster
 
 requires:
-  kafka-client:
+  kafka-client-consumer:
     interface: kafka_client
+  kafka-client-producer:
+    interface: kafka_client
+  kafka-client-admin:
+    interface: kafka_client
```

Comment:
Could this be explained to me? I might be missing context.

Why are there 3 relations as opposed to the existing `extra-user-roles` field? 
I would also add `optional: true` for all of them to avoid annoying log messages.

Reply:
Nvm, this is an app-charm convenience for integration tests, all good, great work.